doc/espresso.adoc

```diff
@@ -0,0 +1,232 @@
+  is *-S0* which seems to give the best results for the cost.
+*-Dpairall*::
+  Minimize the function with all possible assignments of inputs to
+  two-bit decoders. The option *-S1* will perform an exact minimization
+  for each assignment of inputs to decoders, and the option *-S2* will
+  perform an output-phase assignment for each assignment of inputs to
+  decoders. Be warned that pairall requires an exponential number of
+  minimizations !
+*-Dseparate*::
+  Remove the don't-care set from the ON-set of the function.
+*-Dso*::
+  Minimize each function one at a time as a single-output function.
+  Terms will not be shared among the functions. The option *-S1* will
+  perform an exact minimization for each single-output function.
+*-Dso_both*::
+  Minimize each function one at a time as a single-output function, but
+  choose the function or its complement based on which has fewer terms.
+  The option *-S1* will perform an exact minimization for each
+  single-output function and its complement to determine which has fewer
+  terms.
+*-Dstats*::
+  Provide simple statistics on the size of the function.
+*-Dverify*::
+  Checks for Boolean equivalence of two PLA's. Reads two filenames from
+  the command line, each containing a single PLA.
+*-DPLAverify*::
+  Checks for Boolean equivalence of two PLA's by first permuting the
+  columns based on the user supplied variable names. Reads two filenames
+  from the command line.
+*-eeat*::
+  Normally comments are echoed from the input file to the output file.
+  This options discards any comments in the input file.
+*-efast*::
+  Stop after the first EXPAND and IRREDUNDANT operations (i.e., do not
+  iterate over the solution).
+*-ekiss*::
+  Sets up a _kiss_-style minimization problem. This is a hack.
+*-eness*::
+  Essential primes will not be detected.
+*-enirr*::
+  The result will not necessarily be made irredundant in the final step
+  which removes redundant literals.
+*-enunwrap*::
+  The ON-set will not be unwrapped before beginning the minimization.
+*-eonset*::
+  Recompute the ON-set before the minimization. Useful when the PLA has
+  a large number of product terms (e.g., an exhaustive list of
+  minterms).
+*-epos*::
+  Swaps the ON-set and OFF-set of the function after reading the
+  function. This can be used to minimize the OFF-set of a function.
+  _.phase_ (see espresso(5)) in the input file can also specify an
+  arbitrary choice of output phases.
+*-estrong*::
+  Uses the alternate strategy SUPER_GASP (as a replacement for LAST_
+  GASP) which is more expensive, but occasionally provides better
+  results.
+*-o[type]*::
+  Selects the output format. By default, only the ON-set (i.e., type f)
+  is output after the minimization. [type] can be one of *f*, *d*, *r*,
+  *fd*, *dr*, *fr*, or *fdr* to select any combination of the ON-set
+  (f), the OFF-set (r) or the DC-set (d). [type] may also be *eqntott*
+  to output algebraic equations acceptable to _eqntott_(1OCTTOOLS), or
```

Comment:
Like here, referring to something that is not provided by this repo?

Reply:
And `1OCTTOOLS` is not a valid man chapter.

Reply:
You can gather all reviews, and let's schedule a meeting to discuss these.